Library/Homebrew/utils/curl.rb

```diff
@@ -118,18 +118,10 @@ def curl_download(*args, to: nil, partial: true, **options)
            destination.size == %r{^.*Content-Range: bytes \d+-\d+/(\d+)\r\n.*$}m.match(headers)&.[](1)&.to_i
           return # We've already downloaded all the bytes
         end
-      else
-        supports_partial_download = false
-      end
-
-      continue_at = if destination.exist? && supports_partial_download
-        "-"
-      else
-        0
       end
 
       curl(
-        "--location", "--remote-time", "--continue-at", continue_at.to_s, "--output", destination, *args, **options
```

Comment:
CC @reitermarkus for thoughts here. It feels like a bit of a sledgehammer to remove this flag with no user complaints after a flaky/failing test on our end.